nixos/tests/nextcloud/basic.nix

```diff
@@ -14,12 +14,12 @@ in {
     client = { ... }: {
       services.davfs2.enable = true;
       system.activationScripts.davfs2-secrets = ''
-        echo "http://nextcloud/remote.php/webdav/ ${adminuser} ${adminpass}" > /tmp/davfs2-secrets
+        echo "http://nextcloud/remote.php/dav/files/${adminuser} ${adminuser} ${adminpass}" > /tmp/davfs2-secrets
```

Comment:
I understand that this is apparently needed to fix the test. But please leave an explanation in the PR's op and the commit message on the details (i.e. what in rclone has changed that this became necessary)?

Reply:
I think other people described this issue in https://github.com/rclone/rclone/issues/7103

Reply:
Yeah, a tldr (i.e. one short sentence) and a link to the upstream issue is perfectly sufficient IMHO.

Reply:
I'm not sure but I wanted to chime in that there seems to be some issues with Nextcloud and the current released rclone beta. Hopefully this PR is aiming to fix that. I'll move files on the Nextcloud backend, and running rclone lsd doesn't seem to reflect those changes to where the files are located, yet the backend confirms they have been moved.